src/DocumentationBrowserViewExtension/DocumentationBrowserView.xaml.cs

```diff
@@ -26,6 +26,8 @@ public partial class DocumentationBrowserView : UserControl, IDisposable
 
         internal string WebBrowserUserDataFolder { get; set; }
         internal string FallbackDirectoryName { get; set; }
+        //This folder will be used to store all the images located in Degub/en-US/fallback_docs so we don't need to copy all the images per each language
```

Comment:
typo `Degub`, also can we remove the word ?

Reply:
yeah, shouldn't this be `located in /root/en-us/fallback_docs` ? Dynamo will never be deployed in a folder aclled debug or release etc.

Reply:
comments updated, see commit:
https://github.com/DynamoDS/Dynamo/pull/14429/commits/ef88df024b2dee733e26ff26f7c81188bdab999f